ReactWindows/ReactNative.Shared/Common/AccessibilityHelper.cs

```diff
@@ -0,0 +1,229 @@
+using ReactNative.UIManager;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace ReactNative.Common
+{
+    /// <summary>
+    /// Accessibility helper methods.
+    /// </summary>
+    public static class AccessibilityHelper
+    {
+        #region ImportantForAccessibility
+
+        /// <summary>
+        /// Sets the ImportantForAccessibility property for uiElement.
+        /// It uses AutomationProperties.AccessibilityView to implement the element and its children
+        /// visibility to narrator. ImportantForAccessibility value is stored as an attached property.
+        /// </summary>
+        /// <param name="uiElement"></param>
+        /// <param name="importantForAccessibilityNew"></param>
+        public static void SetImportantForAccessibility(UIElement uiElement, ImportantForAccessibility importantForAccessibilityNew)
+        {
+            var importantForAccessibilityOld = GetImportantForAccessibilityAttached(uiElement);
+            if (importantForAccessibilityOld == importantForAccessibilityNew)
+            {
+                return;
+            }
+
+            SetImportantForAccessibilityAttached(uiElement, importantForAccessibilityNew);
+
+            var uiElementAutomationPeer = FrameworkElementAutomationPeer.FromElement(uiElement);
+            // If this element is 'hidden' by a parrent, do not update AccessibilityView property.
+            if (IsHiddenElement(uiElementAutomationPeer))
+            {
+                return;
+            }
+
+            // Update AccessibilityView property accordingly.
+            switch (importantForAccessibilityNew)
+            {
+                case ImportantForAccessibility.Auto:
+                    uiElement.ClearValue(AutomationProperties.AccessibilityViewProperty);
```

Comment:
>ClearValue [](start = 30, length = 10)

the fact that we use Clear here relies on the fact that default value for the attached property is Auto. this is fragile since changing default of attached property will probably not make anyone to look through code to find usages of ClearValue that may be affected. Maybe we can add comments here and also in attached property registration about this?